internal/cmd/export.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/simplesurance/dependencies-tool/internal/deps"
+)
+
+const exportCmdShortHelp = "Generate a dependency tree and export it to a file."
```

Comment:
Do you plan to support several formats? Otherwise I would suggest to explicit mention what's the exported format:

```suggestion
const exportCmdShortHelp = "Generate a json dependency tree and export it to a file."
```